README.md

```diff
@@ -1 +1,50 @@
+
+Therefore, being able to predict purchasing intentions can be a valuable tool because it could potentially allow businesses to implement measures to better capture users with purchasing intention and thus try to convert them into actual revenue generators. Predicting purchasing intentions can also be useful for making more accurate forecasts about the business.
+
+The data set used in this project is the "Online Shoppers Purchasing Intention" dataset provided by the [Gözalan Group](http://www.gozalangroup.com.tr/) and used by Sakar, OC et al. in their 2018 analysis publsihed in [Neural Computing and Applications](https://link.springer.com/article/10.1007/s00521-018-3523-0). The data set was sourced from UCI's Machine Learning Repository (Dua and Graff 2020) at this [link](https://archive.ics.uci.edu/ml/datasets/Online+Shoppers+Purchasing+Intention+Dataset). The specific file used for the analysis found [here](https://archive.ics.uci.edu/ml/machine-learning-databases/00468/online_shoppers_intention.csv). Each row in the data set contains pageview and session information belonging to a different visitor that browsed [Columbia](https://www.columbia.com.tr), an online e-commerce platform based in Turkey. Each row also contains the target class `REVENUE`, a boolean flag indicating whether that user session contributed to revenue or not. Pageview data includes the type of pages that the visitor browsed to and the duration spent on each page. Session information includes visitor-identifying data such as browser information, operating system information as well visitor location and type.
+
+To answer the predictive question stated above, we plan to build a predictive classification model. The data will be split into a training set and a test set based on a 75% and 25% split, respectively. Exploratory Data Analysis (EDA) will be conducted to explore distribution of features and the target class. Feature distributions will be visualized as density plots color-coded by the target class. Target class distribution will be presented as a table showing counts for each class.
+
+Considering this is a binary classification problem, Support Vector Machines (SVM) can be used as the classification algorithm. The hyperparameter k, which indicates the number of nearest neighbors that will be used in the prediction, will be optimized using a cross-validation workflow. The number of folds used will be 10 considering we have a moderately-sized data set of about 12,000 observations. The scoring measure, which will be used to chose optimal k, will be determined based on the results of the EDA and the presence, or lack thereof, of class imbalance.
```

Comment:
I thought we were going to try a few models and compare their accuracy in the project?

Reply:
I am ok with trying a few models if we can - I was not sure if we are allowed to do that since the example breast cancer predictor project only seemed to list 1 algorithm. 

If we all agree, I can add a sentence indicating that this is the approach we will take. We should probably still name the algorithms through. For a project like this I would think: Decision Tree Classifier, Support Vector Machines, K-Nn are good options. Do you have other suggestions?

Reply:
I think to be safe, we can state that "Our main model will be SVM". However, we also plan to explore other models such as Decision Tree, Knn if time allows

Reply:
Ok I will make the adjustment. And I think you are correct Vu regarding k being for Knn anyway!